aries-site/src/data/wcag/components.json

```diff
@@ -48,5 +48,419 @@
+        "label": "Timing Adjustable",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.2",
+        "label": "Pause, Stop, Hide",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.3",
+        "label": "No Timing",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.4",
+        "label": "Interruptions",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.5",
+        "label": "Re-authenticating",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.6",
+        "label": "Timeouts",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.3.1",
+        "label": "Three Flashes or Below Threshold",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.3.2",
+        "label": "Three Flashes",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.3.3",
+        "label": "Animation from Interactions",
+        "status": "not-applicable",
+        "notes": "I don't think hover elevation is considered here. Check with team."
+      },
+      {
+        "rule": "2.4.1",
+        "label": "Bypass Blocks",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.4.2",
+        "label": "Page Titled",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.4.3",
+        "label": "Focus Order",
+        "status": "conditional",
+        "notes": "Depending on context, a proper role of `listitem` or wrapping in an <li> may be needed."
+      },
+      {
+        "rule": "2.4.4",
+        "label": "Link Purpose (In Context)",
+        "status": "conditional",
+        "notes": "Related cards sharing a common call to action such as 'Learn More' may need to be differentiated by using `aria-label` to provide context as to answer 'learn more about what?'."
+      },
+      {
+        "rule": "2.4.5",
+        "label": "Multiple Ways",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.4.6",
+        "label": "Headings and Labels",
+        "status": "conditional",
+        "notes": "Proper semantic headings are needed. Cards marked up as <section> may need an aria-labelledby as well. Cards marked up as <article> will use the first heading within or `aria-label`."
+      },
+      {
+        "rule": "2.4.7",
+        "label": "Focus Visible",
+        "status": "passed",
+        "notes": "When the card is interactive, focus is visible on hover and keyboard focus."
+      },
+      {
+        "rule": "2.4.8",
+        "label": "Location",
+        "status": "not yet evaluated"
+      },
+      {
+        "rule": "2.4.9",
+        "label": "Link Purpose (Link Only)",
+        "status": "conditional",
+        "notes": "Related cards sharing a common call to action such as 'Learn More' may need to be differentiated by using `aria-label` to provide context as to answer 'learn more about what?'."
```

Comment:
For this rule having an aria-label wouldn't be enough in the case where we have links with the text, 'learn more'. This rule requires that the link be identifiable from link text alone.

Reply:
https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-link-only#sufficient

Reply:
Thanks for flagging, I misunderstood when I read through previously. Agree aria-label is sufficient